dev-tools/src/main/resources/plugin-metadata/plugin-descriptor.properties

```diff
@@ -34,6 +34,8 @@
 description=${project.description}
 #
 # 'version': plugin's version
+# version string must be a sequence of nonnegative decimal integers
+# separated by "."'s and may have leading zeros
 version=${project.version}
```

Comment:
Yeah, you're absolutely right. I put it in the wrong place. I'll push a fix.

The only issue that I have with saying it's the same format as the `java.specification.version` is that we're just making people chase down what the specification for `java.specification.version` is.